src/gui/qgsfieldmappingmodel.h

```diff
@@ -92,6 +93,9 @@ class GUI_EXPORT QgsFieldMappingModel: public QAbstractTableModel
     //! Returns a static map of supported data types
     static const QMap<QVariant::Type, QString> dataTypes();
 
+    //! Returns a static map of supported data types
+    static const QList<QgsVectorDataProvider::NativeType> dataTypesV2();
```

Comment:
Should V1 be deprecated? Can we get around the V2 suffix (e.g. `nativeType()`)? Can we document the diff to `dataTypes()`? And can we have a since version?